packages/rules/src/standardTsconfig.ts

```diff
@@ -79,15 +80,20 @@ function getGenerator(context: Context, opts: Options) {
     const fullPath = path.resolve(workspacePackageDir, opts.templateFile);
     const template = JSON.parse(readFileSync(fullPath, "utf-8"));
 
-    return makeGenerator(template, opts.excludedReferences, opts.tsconfigReferenceFile);
+    return makeGenerator(template, opts.excludedReferences, opts.additionalReferences, opts.tsconfigReferenceFile);
   } else if (opts.template) {
-    return makeGenerator(opts.template, opts.excludedReferences, opts.tsconfigReferenceFile);
+    return makeGenerator(opts.template, opts.excludedReferences, opts.additionalReferences, opts.tsconfigReferenceFile);
   } else {
     throw new Error("Unable to make generator");
   }
 }
 
-function makeGenerator(template: any, excludedReferences: ReadonlyArray<string> = [], tsconfigReferenceFile?: string) {
+function makeGenerator(
+  template: any,
+  excludedReferences: ReadonlyArray<string> = [],
+  additionalReferences: ReadonlyArray<string> = [],
+  tsconfigReferenceFile?: string
+) {
   return function generator(context: Context) {
     template = {
       ...template,
```

Comment:
I suggested the `additionalReferences` config, but another option would be to not overwrite existing `references` from the template. At the moment it's reset to an empty array.

```ts
template = {
  ...template,
  references: [],
};
```

Reply:
Noticed that also, might be useful to add a comment if we don't plan to change this